### PR TITLE
closes #12

### LINKED ISSUE #12: end_date >= start_date
https://github.com/degauss-org/dht/blob/4c616cd9f882dc70b81b1738cd1b64bf3f7b187c/R/check_dates.R#L65

### PR DESCRIPTION
**Pull request by erikarasnick**

### CODE REVIEW

File: tests/testthat/test-check_dates.R [hunk 1]
```diff
@@ -49,5 +49,13 @@ test_that("check end date after start date", {
   )
 })
 
+test_that("check end date after start date if end = start", {
+  start_date <- check_dates(c('1/1/21'))
+  end_date <- check_dates(c('1/1/21'))
+  expect_error(
```
Comment (cole-brokamp) [inline tests/testthat/test-check_dates.R:55]: this should not error, right? aren't we expecting success here?

**Review by cole-brokamp — CHANGES_REQUESTED** [review]

File: tests/testthat/test-check_dates.R (hunk 1 above)
Comment (erikarasnick) [reply tests/testthat/test-check_dates.R]: oops, copy paste error. thanks for catching!

**Comment by cole-brokamp** [comment]
Are we running tests in the check?  This test should have failed right?  I cloned and there are several failing tests, but not sure why its not showing up in the R CMD check?

**Comment by erikarasnick** [comment]
Hmm, yes the check should have failed... I assumed tests were running with R CMD check (usually using testthat does this...), but I just looked and I can't see where that's happening 😬

**Comment by erikarasnick** [comment]
I manually went through and ran all the tests (and fixed them to pass... I think I had not updated tests since updating some of the functions, so the code was running as expected but the tests were written incorrectly). Still working on why the RMD check is not running the tests.

**Comment by cole-brokamp** [comment]
I think something weird is going on with the testing structure?  Running tests works for me locally (although I get a lot of failing tests), but the tests are not run during a R CMD Check or a `devtools::check()`

**Comment by erikarasnick** [comment]
Yes, I think so too... it says no tests found

**Comment by erikarasnick** [comment]
I had the tests folder in Rbuildignore 🤦‍♀️

**Review by cole-brokamp — APPROVED** [review]

**Pull request merged**